agent/src/main/java/org/openjdk/jmc/agent/impl/DefaultTransformRegistry.java

```diff
@@ -456,6 +456,10 @@ public String toString() {
 		return classNames;
 	}
 
+	public List<String> getClassNames() {
+		return new ArrayList<>(transformData.keySet());
```

Comment:
This data comes from a set. Any reason for it to be made into a list (implies order, where there is none)? Wrap in Collections#unmodifiableSet?

Reply:
You are correct I have fixed this in https://github.com/openjdk/jmc/pull/60/commits/6dfb0280fa78826ae655044f86cd0e60b97c2a01